packages/contracts/contracts/core/DAO.sol

```diff
@@ -62,33 +60,26 @@ contract DAO is IDAO, Initializable, UUPSUpgradeable, ACL, ERC1271, AdaptiveERC1
         _registerStandard(type(ERC1271).interfaceId);
 
         _setMetadata(_metadata);
-        _setTrustedForwarder(_forwarder);
+        _setTrustedForwarderWithEvent(_forwarder);
```

Comment:
It does more than just setting the forwarder and we did this already in `MetaTxComponent`.

Reply:
https://github.com/aragon/core/blob/ea12ec76f174e850b6ca1470ce3eeca7999c8e83/packages/contracts/contracts/core/component/MetaTxComponent.sol#L41-L51

Reply:
Then, we need to change this in MetaTxComponent.

Not a good practice to include `withEvent` in the name of the function. Just because it also emits the event doesn't mean it should be included in the name..

Doing just `setTrustedForwader` is enough, otherwise, all functions should be renamed that emits events and we should be adding `withEvent` with all of them.

Reply:
I also don't like that 
https://github.com/aragon/core/blob/ea12ec76f174e850b6ca1470ce3eeca7999c8e83/packages/contracts/contracts/core/DAO.sol#L209-L211
doesn't set anything, but just emits an event.

Reply:
it's an internal function and considered a good practice.